content/en/logs/error_tracking/backend.md

```diff
@@ -256,12 +256,12 @@ config.lograge.logger = ActiveSupport::Logger.new(Rails.root.join('log', "#{Rail
 config.lograge.custom_options = lambda do |event|
     {
       error: {
-        type: event.payload[:exception][0],
+        kind: event.payload[:exception][0],
         message: event.payload[:exception][1],
-        stack: event.payload[:exception_object].backtrace
+        stack: event.payload[:exception_object].backtrace.join("\n")
```

Comment:
Stack needs to be a string and backtrace is a list, therefore, the join method call.